app/code/Magento/Sales/Controller/Adminhtml/Order/Creditmemo/Save.php

```diff
@@ -33,21 +33,29 @@ class Save extends \Magento\Backend\App\Action
      */
     protected $resultForwardFactory;
 
+    /**
+     * @var \Psr\Log\LoggerInterface
+     */
+    protected $logger;
+
     /**
      * @param Action\Context $context
      * @param \Magento\Sales\Controller\Adminhtml\Order\CreditmemoLoader $creditmemoLoader
      * @param CreditmemoSender $creditmemoSender
      * @param \Magento\Backend\Model\View\Result\ForwardFactory $resultForwardFactory
+     * @param \Psr\Log\LoggerInterface $logger
      */
     public function __construct(
         Action\Context $context,
         \Magento\Sales\Controller\Adminhtml\Order\CreditmemoLoader $creditmemoLoader,
         CreditmemoSender $creditmemoSender,
-        \Magento\Backend\Model\View\Result\ForwardFactory $resultForwardFactory
+        \Magento\Backend\Model\View\Result\ForwardFactory $resultForwardFactory,
+        \Psr\Log\LoggerInterface $logger
```

Comment:
It is Major Pull Request pls change the taget branch or 

`\Psr\Log\LoggerInterface $logger = null`

https://devdocs.magento.com/guides/v2.2/extension-dev-guide/versioning/codebase-changes.html
New required constructor scalar argument (without pre-configured value)